dts/arm/atmel/sam3x.dtsi

```diff
@@ -262,6 +263,15 @@
 			alarms-count = <1>;
 			status = "disabled";
 		};
+
+		rtt: rtt@400e1a30 {
+			compatible = "atmel,sam-rtt";
+			reg = <0x400e1a30 0x20>;
+			interrupts = <3 0>;
+			clocks = <&pmc PMC_TYPE_PERIPHERAL 3>;
+			source = <ATMEL_SAM_RTT_SOURCE_RTC>;
```

Comment:
The clock management for the Atmel SAM series could use some love.
IMO this is taking a step back using `source` here as a simple enumeration instead of a clock reference.